play-ws-standalone/src/main/java/play/libs/ws/StandaloneWSResponse.java

```diff
@@ -122,6 +122,15 @@
      *
      * Note that this is only usable with a streaming request:
      *
+     * <pre>
+     * {@code
+     * wsClient.url("https://playframework.com")
+     *         .stream() // this returns a CompletionStage<StandaloneWSResponse>
+     *         .thenApply(StandaloneWSResponse::getBodyAsSource);
+     * }
+     * </pre>
      */
-    Source<ByteString, ?> getBodyAsSource();
+    default Source<ByteString, ?> getBodyAsSource() {
+        return Source.single(getBodyAsBytes());
```

Comment:
Yeah, the proper API evolution, in my opinion, is to have this method only for `StreamedResponse` instead of having it in here.